content/docs/command-reference/remote/modify.md

```diff
@@ -185,12 +185,21 @@ parameters to customize the authentication method:
   $ dvc remote modify myremote use_ssl false
   ```
 
-- `ssl_verify` - whether or not to verify SSL certificates. By default SSL
-  certificates are verified.
+- `ssl_verify` - whether or not to verify SSL certificates, or a path to a 
+  custom CA bundle. By default SSL certificates are verified using the CA bundle
+  specified in the AWS config, if one is specified there. If one is not specified,
+  the default CA bundle used by `botocore` will be used. Setting to `false` will 
+  still use SSL if `use_ssl` is `true`, but SSL certificates will not be verified.
 
   ```dvc
   $ dvc remote modify myremote ssl_verify false
   ```
+  
+  To specify a custom CA bundle:
+  
+  ```dvc
+  $ dvc remote modify myremote ssl_verify path/to/cabundle.pem
+  ```
```

Comment:
```suggestion
- `ssl_verify` - whether or not to verify SSL certificates, or a path to a
  custom CA certificates bundle to do so (implies `true`). The CA specified in
  [AWS CLI config](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-settings)
  (if any) is used by default.

  ```dvc
  $ dvc remote modify myremote ssl_verify false
  # or
  $ dvc remote modify myremote ssl_verify path/to/ca_bundle.pem
  ```
```

Reply:
I removed "If one is not specified, the default CA bundle used by `botocore` will be used," because I don't know what it means. Can you clarify this part please?

Reply:
Yeah, it means that if `ssl_verify` isn't set and there is no `ca_bundle` specified in the AWS config, then it falls back to the default CA bundle that `botocore` uses. It reflects the docs in `botocore` on the `verify` parameter:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html

Reply:
OK I see, thanks for the reference. The boto3 ref doesn't indicate what certs it uses so I think it's safe to leave this out because it's implied that `true` will verify SSL with some generic CA.